.github/workflows/security_scan.yaml

```
@@ -0,0 +1,21 @@
+    permissions:
+      contents: read
+      security-events: write
+    environment:
+      name: test-confluencer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/ruby@v1.1.0
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)